core/server/src/test/java/alluxio/master/file/PermissionCheckTest.java

```diff
@@ -465,7 +465,7 @@ public void readDirFailTest() throws Exception {
   public void setStateSuccessTest() throws Exception {
     // set unmask
     Configuration conf = MasterContext.getConf();
-    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSIONS_UMASK, "044");
+    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSIONS_UMASK, "000");
```

Comment:
The change happens in https://github.com/Alluxio/alluxio/pull/2688/files, where we indeed add permission checking on getFileInfo (in master branch). Is this change (044 -> 000) necessary to get tests pass in branch 1.0?